docker/prod/django/prod.Dockerfile

```diff
@@ -12,7 +12,7 @@ RUN --mount=type=cache,target=/var/cache/apk \
 RUN --mount=type=cache,target=/root/.cache/uv \
     --mount=type=bind,source=uv.lock,target=uv.lock \
     --mount=type=bind,source=pyproject.toml,target=pyproject.toml \
-    uv sync --frozen
+    uv sync --frozen --no-dev
```

Comment:
this param disallows uv to instal development dependencies.

Reply:
will be `--extra dev` when using optional dependencies